.circleci/test-deploy.yml

```diff
@@ -5,15 +5,24 @@ orbs:
 filters: &filters
   tags:
     only: /.*/
-jobs:
-  cypress/run:
 
 workflows:
   test-deploy:
     jobs:
       # Make sure to include "filters: *filters" in every test job you want to run as part of your deployment.
       - cypress/run:
           filters: *filters
+          working-directory: examples/simple
```

Comment:
Maybe now is a good time to rename the projects for clarity. Maybe something like:

- `npm-install`
- `yarn-install`
- `custom-install`